src/players/services/players.service.ts

```diff
@@ -130,7 +130,9 @@ export class PlayersService implements OnModuleInit {
         etf2lProfile.bans?.filter((ban) => ban.end > Date.now() / 1000).length >
         0
       ) {
-        throw new AccountBannedError('this account is banned on ETF2L');
+        throw new AccountBannedError(
+          `account ${steamProfile.id} is banned on ETF2L`,
```

Comment:
Instead of using error string link that here, use a `steamId` property in the `AccountBannedError`. See [InsifficientTf2InGameHoursError](https://github.com/tf2pickup-org/server/blob/master/src/players/errors/insufficient-tf2-in-game-hours.error.ts).

Reply:
Got it, thought about it but didn't seem like something I'd do without some feedback on.